src/loaders/core_text.rs

```diff
@@ -550,16 +550,21 @@ impl Font {
         let core_graphics_size = CGSize::new(canvas.size.x() as f64, canvas.size.y() as f64);
         core_graphics_context.fill_rect(CGRect::new(&CG_ZERO_POINT, &core_graphics_size));
 
-        match rasterization_options {
-            RasterizationOptions::Bilevel => {
+        match rasterization_options.antialiasing_strategy {
+            AntialiasingStrategy::Bilevel => {
                 core_graphics_context.set_allows_font_smoothing(false);
                 core_graphics_context.set_should_smooth_fonts(false);
                 core_graphics_context.set_should_antialias(false);
             }
-            RasterizationOptions::GrayscaleAa | RasterizationOptions::SubpixelAa => {
+            AntialiasingStrategy::GrayscaleAa | AntialiasingStrategy::SubpixelAa => {
                 // FIXME(pcwalton): These shouldn't be handled the same!
+                if rasterization_options.use_thin_strokes {
+                    core_graphics_context.set_should_smooth_fonts(false);
+                    core_graphics_context.set_font_smoothing_style(16);
```

Comment:
nit: can you add documentation of how you derive this? At the very minimum it may be useful to reference how WebCore (part of WebKit) defines this: https://cs.github.com/ur0/webkit-source/blob/474163e70412a2f0ff3a80b961dea1bb9e000041/WebCore/platform/graphics/win/FontCGWin.cpp?q=%22kCGFontSmoothingStyleMinimum%22#L218

Reply:
Got rid of this line; it doesn't seem to affect anything unless using subpixel-antialiased fonts (which aren't enabled in almost all versions of MacOS that we target).